lib/phobos_db_checkpoint/handler.rb

```diff
@@ -14,6 +15,19 @@ module ClassMethods
       include Phobos::Instrumentation
       include Phobos::Handler::ClassMethods
 
+      # Name...
+      # => retry_consume?
+      # => back_off?
+      # => consume_again?
+      # => fail_consume?
+      # => fail_permanently?
+      # => give_up?
+      # => stop_message?
+      # => consume_failure_condition_met?
+      def retry_consume?(event, event_metadata, exception)
```

Comment:
Actually I think this name is kind of decent now. Read it from a readme point of view, or a KlearSearch point of view, it seems like a name that makes sense